### PR TITLE
updated vision statement for elaboration2

### PR DESCRIPTION
**Pull request by kny4**
Alright, team. Here's the updated Vision Statement. It's kinda a lot different, but this has also kinda been the vision for the vision statement (whoa) the entire time. Let me know what you think! 
P.S. I created a new label just called update, so enjoy that.

### CODE REVIEW

**Comment by kny4** [comment]
I updated the Domain Model link in the Readme file too, so it should actually take you there now. It wasn't working because it was still called "classDiagram" in the link.

**Review by andrewJamesBaker — APPROVED** [review]
Looks good to me. Good work.

**Comment by kvlinden** [comment]
I like this rewrite. 
But I'd say that it's odd that you only mention anchoring once, even though it's the key to starting new habits. Perhaps you could reiterate that word in point 2 (make it easy), which, I think, is the basis of your support for anchoring.
Also, doesn't J. Clear suggest buddies/accountability as another key factor? If so, I'd mention buddies in that first paragraph as well.
Finally, that last sentence should be a separate paragraph, after the enumerated list.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+  1. Make it obvious. Habit Buddy makes your habit obvious by sending reminders and giving you the opportunity to contact a Buddy. After filling out a simple profile, you’ll be able to easily set up accountability with your Buddy and share your habit goal with each other. 
+  2. Make it easy. Habit Buddy makes your habit easy by helping you associate your new habit with something you already do and enjoy. You’ll choose an activity that you enjoy doing (i.e. drinking coffee at 8am) and commit to practicing your new habit afterward (i.e. doing yoga for 30 minutes).
+  3. Make it attractive. Habit Buddy makes your habit attractive by matching you with a Buddy who either shares similar goals and interests with you or who already practices your desired habit regularly.
```
Comment (zeph01) [inline README.md:9]: Make it attractive. We all tend to adopt the habits that are praised and approved by our culture. Habit Buddy makes your habit attractive by allowing you to join a culture where your desired habit is already a normal habit and is praised and respected.  It matches you with a Buddy...

File: README.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+ 
+Rooted in these ideas, our mobile app, Habit Buddy, a visionary project to help you start a new habit. By matching you to someone with similar goals and interests, Habit Buddy sets you both up for success and reward by giving you the means to create accountability and encourage each other as you work towards your personal and shared goals. Habit Buddy makes it easy and fun to start and maintain a habit by following Clear’s four laws: 
+  1. Make it obvious. Habit Buddy makes your habit obvious by sending reminders and giving you the opportunity to contact a Buddy. After filling out a simple profile, you’ll be able to easily set up accountability with your Buddy and share your habit goal with each other. 
+  2. Make it easy. Habit Buddy makes your habit easy by helping you associate your new habit with something you already do and enjoy. You’ll choose an activity that you enjoy doing (i.e. drinking coffee at 8am) and commit to practicing your new habit afterward (i.e. doing yoga for 30 minutes).
```
Comment (zeph01) [inline README.md:8]: Make it easy. **Doing things that makes you happy/gives you positive feeling is easy to do. HabitBuddy helps you create a motivation ritual by doing something you enjoy immediately before or after a difficult habit.** You'll choose an activity...  

Also.."“The Diderot Effect states that obtaining a new possession often creates a spiral of consumption that leads to additional purchases.....One of the best ways to build a new habit is to identify a current habit you already do each day and then stack your new behavior on top. This is called habit stacking.” Pulled this from the AtomicHabits book. Could use to describe Habit Stacking. Not sure if this is important in vision statement though. Probably in the about or info page of the app.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+Good habits are difficult to start and difficult to keep. Even the most diligent among us struggle to maintain healthy habits. According to James Clear's book, *Atomic Habits*, one of the most effective ways to start a new habit is to anchor it to an activity or interest you already do on a regular basis, which he lays out in detail through four laws: make it obvious, make it easy, make it attractive, and make it satisfying. 
+ 
+Rooted in these ideas, our mobile app, Habit Buddy, a visionary project to help you start a new habit. By matching you to someone with similar goals and interests, Habit Buddy sets you both up for success and reward by giving you the means to create accountability and encourage each other as you work towards your personal and shared goals. Habit Buddy makes it easy and fun to start and maintain a habit by following Clear’s four laws: 
+  1. Make it obvious. Habit Buddy makes your habit obvious by sending reminders and giving you the opportunity to contact a Buddy. After filling out a simple profile, you’ll be able to easily set up accountability with your Buddy and share your habit goal with each other.
```
Comment (zeph01) [inline README.md:7]: Make it obvious. **Small changes in your environment can lead you to make large changes in your lifestyle.** Habit Buddy makes your habit obvious....

File: README.md [hunk 4]
```diff
@@ -1,9 +1,14 @@
...
+  1. Make it obvious. Habit Buddy makes your habit obvious by sending reminders and giving you the opportunity to contact a Buddy. After filling out a simple profile, you’ll be able to easily set up accountability with your Buddy and share your habit goal with each other. 
+  2. Make it easy. Habit Buddy makes your habit easy by helping you associate your new habit with something you already do and enjoy. You’ll choose an activity that you enjoy doing (i.e. drinking coffee at 8am) and commit to practicing your new habit afterward (i.e. doing yoga for 30 minutes).
+  3. Make it attractive. Habit Buddy makes your habit attractive by matching you with a Buddy who either shares similar goals and interests with you or who already practices your desired habit regularly.  
+  4. Make it satisfying. Habit Buddy makes your habit satisfying by tracking your daily progress through a streak count. Buddies can also see each other’s progress and encourage and inspire one another in practicing a new habit.
```
Comment (zeph01) [inline README.md:10]: Make it satisfying. **We like to repeat activities that are satisfying.** Habit Buddy makes your habit satisfying....

**Review by zeph01 — CHANGES_REQUESTED** [review]
I really like the vision statement you have updated! I have made some comments on the file for specific lines. I added descriptions for why the "make it obvious, easy, attractive, and satisfying" are even necessary. It sounded helpful in my mind. But let me know what you think.

I found this website that gives a summary of the book in just several lines for every chapter. I found it helpful to look through it by finding the key word like "obvious" in the website and reading how the book describes it.
https://www.samuelthomasdavies.com/book-summaries/self-help/atomic-habits/

File: README.md (hunk 1 above)
Comment (kny4) [reply README.md:9]: Ooooh "join a culture"! I like it (or I'm attracted to it HAHAHAHA)

File: README.md (hunk 2 above)
Comment (kny4) [reply README.md:8]: We should definitely keep the Diderot Effect for something, but yeah, maybe the vision statement is getting kinda long.

**Comment by kny4** [comment]
Alright people! I've got some new updates, so let me know what you think! @zeph01 @andrewJamesBaker @Bongo9911 @JoePastucha @nstrain @kvlinden

**Review by kvlinden — APPROVED** [review]
I like that much better!

**Pull request merged**